pkg/services/accesscontrol/pluginutils/utils.go

```diff
@@ -0,0 +1,33 @@
+package pluginutils
+
+import (
+	"github.com/grafana/grafana/pkg/plugins"
+	ac "github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+func ToRegistrations(pluginName string, regs []plugins.RoleRegistration) []ac.RoleRegistration {
+	res := make([]ac.RoleRegistration, 0, len(regs))
+	for i := range regs {
+		res = append(res, ac.RoleRegistration{
+			Role: ac.RoleDTO{
+				Version:     1,
```

Comment:
question: is it fine to keep the version number fixed or are you planning to extend this in the future so that a plugin can update its roles if needed?

Reply:
Ah, I see, fixed roles and plugin roles always get updated: https://github.com/grafana/grafana-enterprise/pull/4025/files#diff-a2acf343e76c254f6b7f70f365d6398dccb56652b4c8bdda86ab892bf79353abR259-R260